frontend/tests/acceptance/features/webUIDashboard/notes.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Add notes to job application
+  As a user
+  I want to add notes to my job applications
+  So that I can remark my job applications
+
+  Background: User has access to dashboard
+    Given a user has been registered with the following details:
+      | name | email          | password |
+      | test | test@email.com | testpass |
+    And the user has logged in to the dashboard with email "test@email.com" and password "testpass"
```

Comment:
Maybe you can do regular login in one step and then navigate to dashboard in another step? :thinking: IMO it would look better.